apps/web/views.py

```diff
@@ -35,29 +35,35 @@ def internal_server_error(request):
 
 
 @throttle_classes([AnonRateThrottle, ])
-@api_view(['POST', ])
+@api_view(['GET', 'POST'])
 @permission_classes((permissions.AllowAny,))
 def contact_us(request):
     try:
         user = User.objects.get(username=request.user)
         name = user.username
         email = user.email
-        request_data = {"name": name, "email": email}
-        request_data['message'] = request.data['message']
-        serializer = ContactSerializer(data=request_data)
+    except:
+        serializer = ContactSerializer(data=request.data)
```

Comment:
This is redundant. Same thing is happening in `POST` check one just that data is different. I think it can be easily handled with a simple if condition. @RishabhJain2018